content/docs/self-hosted-runners.md

```diff
@@ -437,17 +437,23 @@ provisioned through environment variables instead of files.
 
 The `cml runner` command can also be used to manually set up a local machine,
 on-premise GPU cluster, or any other cloud compute resource as a self-hosted
-runner. Simply [install CML](/doc/install) and then run:
+runner. Instead of having a `launch-runner` job (as in the YAML above), simply
+[install CML](/doc/install) on the local machine and then run:
 
 ```cli
 $ cml runner launch \
   --repo="$REPOSITORY_URL" \
   --token="$PERSONAL_ACCESS_TOKEN" \
-  --labels="local,runner" \
+  --labels="cml-gpu" \
   --idle-timeout=180
 ```
 
 The machine will listen for jobs from your repository and execute them locally.
+The label(s) may be anything, but at least one of them must match the one
+configured in the `runs_on` of the YAML.
+
+☝️ **Note** The runner will automatically pull the Docker image in the local
+machine and execute the recipe in a newly created container.
```

Comment:
Should we move this note up before the code sample? Also, we can use a proper admon like this:

```suggestion
<admon type="info">

The runner will automatically pull the Docker image in the local
machine and execute the recipe in a newly created container.

</admon>
```

Reply:
> runner will automatically pull the Docker image

a bit misleading IMO. CML runners will only pull a Docker image if you specifically request it (i.e. `image: ...` in your CI workflow). Docker images aren't actually required. You could just run CI jobs in a sandboxed user account on your machine (no Docker required).

```suggestion
<admon type="info">

If your CI workflow uses a Docker `image`, you will need to have Docker installed on
your local machine. The CML runner will automatically pull images onto your local
machine and run workflow in temporary containers.

</admon>
```